config/fxdata/objects.cfg

```diff
@@ -713,3 +713,10 @@ Properties = CHOWNED_ON_ROOM_CLAIM
 Name = SPELLBOOK_POSS
 Genre = SPELLBOOK
 Properties = CHOWNED_ON_ROOM_CLAIM
+
+[object136]
+Name = TEST_OBJ
+Genre = DECORATION
+Properties = CHOWNED_ON_ROOM_CLAIM DESTROYED_ON_ROOM_PLACE
+Image = "test1.tga"
```

Comment:
Your image is displayed the same in first person and top down view. There should be different images for this of course.

Reply:
There is also `Image3d` with `Region3d` for another image. I will update PR

Reply:
done